plugins/woocommerce/includes/shipping/free-shipping/class-wc-shipping-free-shipping.php

```diff
@@ -89,19 +89,19 @@ public function init() {
 	public function init_form_fields() {
 		$this->instance_form_fields = array(
 			'title'            => array(
-				'title'       => __( 'Title', 'woocommerce' ),
+				'title'       => __( 'Name', 'woocommerce' ),
 				'type'        => 'text',
 				'description' => __( 'This controls the title which the user sees during checkout.', 'woocommerce' ),
 				'default'     => $this->method_title,
 				'desc_tip'    => true,
 			),
 			'requires'         => array(
-				'title'   => __( 'Free shipping requires...', 'woocommerce' ),
+				'title'   => __( 'Free shipping requires', 'woocommerce' ),
 				'type'    => 'select',
 				'class'   => 'wc-enhanced-select',
 				'default' => '',
 				'options' => array(
-					''           => __( 'N/A', 'woocommerce' ),
+					''           => __( 'No requirement', 'woocommerce' ),
 					'coupon'     => __( 'A valid free shipping coupon', 'woocommerce' ),
 					'min_amount' => __( 'A minimum order amount', 'woocommerce' ),
 					'either'     => __( 'A minimum order amount OR a coupon', 'woocommerce' ),
```

Comment:
Not related to this PR but just pointing out in the design it's `A minimum order amount OR coupon`.

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/40615/commits/78ecd994be3cf1611b6a5a0e50f428918035d963